### PR TITLE
[TF 2.0 docs] Add example for EarlyStopping

### PR DESCRIPTION
**Pull request by aweers**
I think a short example like this is very useful for user who don't have much experience and want to use a callback for their first time.

### CODE REVIEW

**Comment by aweers** [comment]
If the responses are positive I will create examples for the other callbacks, too. ;)

File: tensorflow/python/keras/callbacks.py [hunk 1]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
           the epoch with the best value of the monitored quantity.
           If False, the model weights obtained at the last step of
           training are used.
+          
+  Example:
+  
+  ```python
+  # first create the callback
```
Comment (kyscg) [inline tensorflow/python/keras/callbacks.py]:
```suggestion
  # Firstly, let's create the callback
```

File: tensorflow/python/keras/callbacks.py [hunk 2]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
...
+  callback = tf.keras.callbacks.EarlyStopping(monitor='val_loss', patience=3)
+  # this callback will stop training when there is no improvement in
```
Comment (kyscg) [inline tensorflow/python/keras/callbacks.py]:
```suggestion
  # This callback will stop training when there is no improvement in
```

File: tensorflow/python/keras/callbacks.py [hunk 3]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
...
+  # validation loss for three epochs
```
Comment (kyscg) [inline tensorflow/python/keras/callbacks.py]:
```suggestion
  # the validation loss for three consecutive epochs.
```

File: tensorflow/python/keras/callbacks.py [hunk 4]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
...
+  # then simply train the model with the callback
```
Comment (kyscg) [inline tensorflow/python/keras/callbacks.py]:
```suggestion
  # We now simply train the model with the callback.
```

**Review by kyscg — CHANGES_REQUESTED** [review]
Great work @aweers 

I tried to make the comments interactive. Change these and keep going

Cheers!!!

**Comment by aweers** [comment]
Thank you again @kyscg! I like the interactive style and will try to make the next examples in a similiar way!

**Review by kyscg — DISMISSED** [review]

**Comment by rthadur** [comment]
@aweers can you please sign CLA

**Comment by aweers** [comment]
@rthadur I signed the CLA ;)

**Comment by kyscg** [comment]
@rthadur @rchao Gentle ping to review

File: tensorflow/python/keras/callbacks.py [hunk 5]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
           the epoch with the best value of the monitored quantity.
           If False, the model weights obtained at the last step of
           training are used.
+          
+  Example:
+  
+  ```python
+  # Firstly, let's create the callback
+  callback = tf.keras.callbacks.EarlyStopping(monitor='val_loss', patience=3)
+  # This callback will stop training when there is no improvement in
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: s/stop training/stop the training

File: tensorflow/python/keras/callbacks.py [hunk 6]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
           the epoch with the best value of the monitored quantity.
           If False, the model weights obtained at the last step of
           training are used.
+          
+  Example:
+  
+  ```python
+  # Firstly, let's create the callback
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: This line of comment is probably self-explanatory by the code.

File: tensorflow/python/keras/callbacks.py [hunk 7]
```diff
@@ -1018,6 +1018,18 @@ class EarlyStopping(Callback):
...
+  callback = tf.keras.callbacks.EarlyStopping(monitor='val_loss', patience=3)
+  # This callback will stop training when there is no improvement in
+  # the validation loss for three consecutive epochs.
+  # We now simply train the model with the callback.
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: This line as well, self-explanatory by model.fit()

**Review by rchao — COMMENTED** [review]
Thanks for the PR! Please see the comments.

**Comment by aweers** [comment]
@rchao I updated the file with your suggestions

**Comment by aweers** [comment]
But somehow I don't see a new commit? Only if I try to edit the file again, I see the changes I made...

**Comment by rchao** [comment]
> But somehow I don't see a new commit? Only if I try to edit the file again, I see the changes I made...

Thanks! I don't see it either. Please let me know once the commit is made.

**Comment by aweers** [comment]
@rchao @rthadur now the commit is made, don't know why it took some time... But CLA check failed again, so it need to be set by you ;)

**Review by rchao — APPROVED** [review]
Looks good! Thank you.
